### PR TITLE
inverted index hotfix

### PR DESCRIPTION
**Pull request by jinhwak11**
Hotfix for loading data 

The issue was I designed that if two more words have the same character at the beginning, it does not read data again and reused the previous one. But I didn't add the data for that case, which generates an error, so I fixed it.

### CODE REVIEW

File: M2/InvertedIndexLoader.py [hunk 1]
```diff
@@ -50,6 +50,6 @@ def getIndexDataAllTokens(tokens):
                 skippointer  = json.loads(l[-1]) # read skippointer first 
             from InvertedIndex import Postings
             data = [Postings.from_json(d) for d in json.loads(l[skippointer[t]])[t]]  # read only part of file where token is 
-            data_dict[t] = data 
-            prevc = currentc
+        data_dict[t] = data
```
Comment (nagrawal63) [inline M2/InvertedIndexLoader.py:53]: Isn't this problematic?
Ex: for search "Design data", the code will store the same `data` variable for the second token as well.

Comment (jinhwak11) [reply M2/InvertedIndexLoader.py:53]: I unindented data part.

Comment (nagrawal63) [reply M2/InvertedIndexLoader.py:53]: Hmm, I think this should work now.

Comment (nagrawal63) [reply M2/InvertedIndexLoader.py:53]: Will merge

**Review by nagrawal63 — APPROVED** [review]
LGTM!

**Pull request merged**